code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -535,6 +535,44 @@ obj/item/asteroid/basilisk_hide/New()
 			C.icon_state = "[initial(C.icon_state)]_goliath[C.hidecount]"
 			C._color = "mining_goliath[C.hidecount]"
 
+/mob/living/simple_animal/hostile/asteroid/goliath/david
+	name = "david"
+	desc = "Goliaths are fiercely protective of their young, threatening them is not advised under any conditions."
```

Comment:
You could add some basic mob AI to attract adult goliaths when the young are attacked.

Reply:
Side note: Comma splice here